packages/flutter/lib/src/widgets/text.dart

```diff
@@ -201,6 +201,77 @@ class DefaultTextStyle extends InheritedTheme {
   }
 }
 
+/// The [TextHeightBehavior] that will apply to descendant [Text] widgets which
+/// have not explicitly set [Text.textHeightBehavior].
```

Comment:
We're going to need to explain how this interacts with DefaultTextStyle

Reply:
Added some docs explaining the relationship.